src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,24 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  query: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList: React.FC<Props> = ({ movies, query }) => {
+  const visibleMovies = movies.filter((movie) => {
```

Comment:
It would be better filter this on the upper lvl, you have `like a state` & query in the upper component